pkg/net/libp2p/key.go

```diff
@@ -49,11 +48,16 @@ func operatorPublicKeyToNetworkPublicKey(
 		return nil, fmt.Errorf("libp2p supports only secp256k1 operator keys")
 	}
 
-	return &libp2pcrypto.Secp256k1PublicKey{
-		Curve: DefaultCurve,
-		X:     operatorPublicKey.X,
-		Y:     operatorPublicKey.Y,
-	}, nil
+	rawOperatorPublicKey := operator.MarshalCompressed(operatorPublicKey)
+
+	secpNetworkPublicKey, err := secp.ParsePubKey(rawOperatorPublicKey)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I think we can still use `btcec` directly:
```suggestion
	operatorPublicKeyBytes := operator.MarshalCompressed(operatorPublicKey)

	btcecPublicKey, err := btcec.ParsePubKey(operatorPublicKeyBytes)
	if err != nil {
		return nil, err
	}
```

Reply:
Done in dbcc2dfc6ab514c423f55f42a04a4baf03395f3c.